### PR TITLE
updated About Me copy and links

### PR DESCRIPTION
**Pull request by jtallen**
still need to add bold styling and link to final part of descriptions - need to talk about this later
Screen Shot 2022-01-11 at 1 23 51 PM

### CODE REVIEW

File: constants/data.js
Comment (cchanningallen) [inline constants/data.js]: Should `REACH OUT` be a mailto link? eg
```suggestion
      `🙋‍♂️ I'm looking for a job! I thrive in environments with humble, curious, kind folks. If you think I might be a fit, please <a href="mailto:jturnerallen@gmail.com"><strong>reach out</strong></a> - I'd love to hear from you.`,
```
(make sure ^that works 😅 )

**Review by cchanningallen — COMMENTED** [review]
Looks great overall! Two quick things:
1. Please TAL at my comment, and verify it works
2. For UI changes, mind adding a screenshot showing that the change did what it's supposed to do?

File: constants/data.js
Comment (jtallen) [reply constants/data.js]: It doesn't work unfortunately. The bold and anchor tags just get displayed on the page as '<a>' etc.

The descriptions are mapped to the middle of <p> tags so I wanted to talk about the best way to go about fixing this.

I have updated the initial PR to include a picture of the working changes.

Comment (cchanningallen) [reply constants/data.js]: Ok, looked through this more thoroughly - since these are just dumped as text into `<p>` tags as strings [here](https://github.com/jtallen/turner-site/blob/about/components/AboutMe.js#L98-L105), you need add the last one directly into the `AboutMe` page. Something like:

```
            {userData.about.description?.map((desc, idx) => (
              <p
                key={idx}
                className="text-xl text-gray-700 mb-4 dark:text-gray-300 "
              >
                {desc}
              </p>
            ))}
            <p
                key={idx}
                className="text-xl text-gray-700 mb-4 dark:text-gray-300 "
              >
                🙋‍♂️ I'm looking for a job! I thrive in environments with humble, curious, kind folks. If you think I might be a fit, please <a href="mailto:jturnerallen@gmail.com"><strong>reach out</strong></a> - I'd love to hear from you.
            </p>
```
lmkwyt!

**Comment by jtallen** [comment]
the description paragraphs should now show anchor/strong tags correctly, they are now hardcoded into the page and removed from the data.js file

edit: the only remaining issue is to discuss how the images are currently hosted for the tech stack, how to recreate that, or whether to scrap it and start over

Screen Shot 2022-01-11 at 1 54 05 PM

**Comment by jtallen** [comment]
**Tech Stack:** removed old/irrelevant icons and added new ones

**Before:**
Screen Shot 2022-01-11 at 3 36 30 PM

**After:**
Screen Shot 2022-01-11 at 3 02 50 PM

File: components/AboutMe.js
Comment (cchanningallen) [inline components/AboutMe.js]: Small nit: wdyta throwing a hover style on this? Super small, but a nice micro-animation can make your site "pop" subtly :)

For example: `hover:border-indigo-500 hover:text-indigo-500 transition-all`

(the `transition-all` just adds a slight transition animation from/to the hover color)

**Review by cchanningallen — APPROVED** [review]
One nit, otherwise - great work! Approved pending that hover animation (your call whether to add or not)

File: components/AboutMe.js
Comment (jtallen) [reply components/AboutMe.js]: Added hover style for all three similar links on this page, also added image hover text for the tech stack icons (will revisit this with a higher quality system later)

**Pull request merged**
